compiler/damlc/tests/BUILD.bazel

```diff
@@ -27,15 +27,17 @@ da_haskell_test(
 # Tests for damlc visual
 da_haskell_test(
```

Comment:
I am not sure we gain much by having this be a separate test suite from the existing shake tests so I would propose that you just move it to that test suite.

Reply:
Moved all this to shake test suite